src/general/PauseMenu.tscn

```diff
@@ -48,67 +48,67 @@ __meta__ = {
 }
 
 [node name="Resume" type="Button" parent="CenterContainer/PrimaryMenu"]
-margin_right = 250.0
+margin_right = 284.0
 margin_bottom = 40.0
 rect_min_size = Vector2( 250, 40 )
-text = "Resume"
+text = "PAUSE_MENU_RESUME"
```

Comment:
I think it would be preferrable to name the strings by what they mean, rather than where they are
So all of the things here would be RESUME_GAME...